src/tracer.js

```diff
@@ -524,17 +533,7 @@ module.exports.sendTraceSync = function sendTraceSync() {
  * @param {string} value value for the added label
  */
 module.exports.label = function addLabel(key, value) {
-    const tracerObj = module.exports.getTrace();
-    if (!tracerObj || !tracerObj.currRunner) {
-        utils.debugLog('Failed to label without an active tracer');
-        return;
-    }
-    let labels = {};
-    labels[key] = value;
-    labels = utils.flatten(labels);
-    Object.keys(labels).forEach((k) => {
-        eventInterface.addLabelToMetadata(tracerObj.currRunner, k, labels[k]);
-    });
+    config.setConfig({...config.getConfig(), labels: [...config.getConfig().labels, {[key]: value}]})
```

Comment:
You've removed the ability to provide complex objects into add label - please return it.